demo/02-customer-infra.sh

```diff
@@ -18,3 +18,29 @@ az network vnet create \
   --subnet-name ${CUSTOMER_VNET_SUBNET1} \
   --subnet-prefixes 10.0.0.0/24 \
   --nsg "${NSG_ID}"  --location ${LOCATION}
+
+az keyvault create \
+  --name "${CUSTOMER_KV_NAME}" \
+  --resource-group "${CUSTOMER_RG_NAME}" \
+  --location "${LOCATION}" \
+  --enable-rbac-authorization true
+
+# Get current user's object ID
+CURRENT_USER_ID=$(az ad signed-in-user show --query id -o tsv)
```

Comment:
We could set it to:

$(az ad signed-in-user show -o json | jq -r '.id')

as done in other places